docs/visualize/vega.asciidoc

```diff
@@ -343,12 +1572,37 @@ additional configuration options.
       controlsLocation: top
       // Can be `vertical` or `horizontal` (default).
       controlsDirection: vertical
-      // If true, hides most of Vega and VegaLite warnings
+      // If true, hides most of Vega and Vega-Lite warnings
       hideWarnings: true
       // Vega renderer to use: `svg` or `canvas` (default)
       renderer: canvas
     }
   }
-  /* the rest of Vega code */
 }
 ----
+
+
+[[vega-notes]]
+[[vega-useful-links]]
+=== Resources and examples
+
+experimental[] To learn more about Vega and Vega-Lite, refer to the resources and examples.
+
+==== Vega editor
+The https://vega.github.io/editor/[Vega Editor] includes examples for Vega & Vega-Lite, but does not support any
+{kib}-specific features like {es} requests and interactive base maps.
+
+==== Vega-Lite resources
+* https://vega.github.io/vega-lite/tutorials/getting_started.html[Tutorials]
+* https://vega.github.io/vega-lite/docs/[Docs]
+* https://vega.github.io/vega-lite/examples/[Examples]
+
+==== Vega resources
+* https://vega.github.io/vega/tutorials/[Tutorials]
+* https://vega.github.io/vega/docs/[Docs]
+* https://vega.github.io/vega/examples/[Examples]
+
+TIP: When you use the examples, you may
```

Comment:
Suggestion: "When you use the examples **in Kibana**, you may ..."

Reply:
Done